### ISSUE TITLE
3d alt-tab effects are bugged

### ISSUE THREAD

**Issue by TPO100Cofficial** [linuxmint/cinnamon#11218]
coverflow and timeline effects don't work properly for me
they're supposed to smoothly scale out the window you're currently focused into the array of all open windows, and scale the one you select back into view
but instead the moment i press alt + tab, the windows are already at the end of the animation
but i don't think it's an intentional change? because they still change the angles when i tab out and in
[video](https://cdn.discordapp.com/attachments/820771886585217094/1017180436331499631/2022-09-07_18-07-08.mp4) of the problem described above 

 * Cinnamon 5.4.12, Mutter 42.2, but this also happened with Cinn 5.4.11
 * Started when I upgraded to Mint 21
 * AMD ATI Radeon Vega Series / Radeon Vega Mobile Series
 * 64 bit

[notably, other people have experienced this](https://forums.linuxmint.com/viewtopic.php?f=208&t=381355&p=2229364#p2229364)

**Comment by mfreeman72** [comment]
Good catch! I hadn't noticed this. The scale-out/in animation is definitely not working anymore.

**Comment by akirapink** [comment]
still not fixed?

**Comment by akirapink** [comment]
bump: it's been 2 years, still not fixed
